src/lib/exportSave.ts

```diff
@@ -29,7 +29,10 @@ const save_ = async (file: ModelingAppFile, toastId: string) => {
           window.electron.join(downloadDir, file.name),
           new Uint8Array(file.contents)
         )
-        toast.success(EXPORT_TOAST_MESSAGES.SUCCESS, { id: toastId })
+        toast.success(EXPORT_TOAST_MESSAGES.SUCCESS + ' [TEST]', {
```

Comment:
@jacebrowning what's the [TEST] suffix about? Was that for local testing?

Reply:
It's a visual indicator that we are hitting this alternative logic for testing that skips the dialog. I'd like to keep it.

Reply:
Ohhh okok I didn't realize this was an IS_PLAYWRIGHT block. Thanks!